examples/gimbal/gimbal.cpp

```diff
@@ -85,81 +85,85 @@ int main(int argc, char** argv)
 
     std::cout << "Start controlling gimbal...\n";
     Gimbal::Result gimbal_result = gimbal.take_control(gimbal_id, Gimbal::ControlMode::Primary);
+    sleep_for(seconds(5));
     if (gimbal_result != Gimbal::Result::Success) {
         std::cerr << "Could not take gimbal control: " << gimbal_result << '\n';
         return 1;
     }
 
+    // Fetch the gimbal attitude
+    auto [result, attitude] = gimbal.get_attitude(gimbal_id);
+
     std::cout << "Use yaw mode to lock to a specific direction...\n";
 
     std::cout << "Look North...\n";
-    gimbal.set_angles(0, 0.0f, 0.0f, 0.0f, Gimbal::GimbalMode::YawLock, Gimbal::SendMode::Once);
-    sleep_for(seconds(2));
+    gimbal.set_angles(gimbal_id, 0.0f, 0.0f, -attitude.euler_angle_north.yaw_deg, Gimbal::GimbalMode::YawLock, Gimbal::SendMode::Once);
```

Comment:
I don't understand the `-attitude.euler_angle_north.yaw_deg` added in these lines. Why is that necessary?

Reply:
@JonasVautherin The initial implementation for Look North/East/South/West was originally designed as Look Forward/Left/Back/Right. Therefore, I had to adjust the yaw based on its current orientation relative to North. If this adjustment isn't necessary, we could consider renaming it to simply Look Forward/Left/Back/Right. However, this might be redundant since this functionality is already included in the code.

Reply:
Right, makes sense!